mindmeld/components/dialogue.py

```diff
@@ -656,6 +658,10 @@ def prompt(self, text):
         self._logger.warning('prompt() is deprecated. Please use reply() and listen() instead')
         self.reply(text)
 
+    def sleep(self, delay=0):
+        """Adds a 'sleep' directive."""
```

Comment:
document the delay argument - see line 655

https://github.com/cisco/mindmeld/pull/15/files#diff-c21e7ac893d7b0e1d68df2df2ce7ff52R655